packages/node-core/src/db/db.module.ts

```diff
@@ -23,6 +32,42 @@ const DEFAULT_DB_OPTION: DbOption = {
   username: process.env.DB_USER ?? 'postgres',
   password: process.env.DB_PASS ?? 'postgres',
   database: process.env.DB_DATABASE ?? 'postgres',
+  ssl: process.env.DB_SSL === 'true' ?? false,
```

Comment:
Rather than having `DB_SSL`, we can use whether `DB_CA_CERT` is defined as the check to use ssl or not.